### PR TITLE
Revert "BXMSPROD-1540 composite actions centralized github actions"

### PR DESCRIPTION
**Pull request by radtriste**
Reverts kiegroup/kogito-runtimes#1692

### CODE REVIEW

File: .github/workflows/kogito-apps-pr.yml [hunk 1]
```diff
@@ -22,33 +22,52 @@ jobs:
...
+      - name: Cache Maven packages
+        uses: actions/cache@v2
         with:
-          annotations-prefix: ${{ runner.os }}-${{ matrix.java-version }}/${{ matrix.maven-version }}
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Build Chain ${{ matrix.java-version }}
+        id: build-chain
+        uses: kiegroup/github-action-build-chain@v2.6.13
```
Comment (radtriste) [inline .github/workflows/kogito-apps-pr.yml:59]:
```suggestion
        uses: kiegroup/github-action-build-chain@v2.6.18
```

File: .github/workflows/kogito-examples-pr.yml [hunk 2]
```diff
@@ -22,31 +22,51 @@ jobs:
...
+      - name: Cache Maven packages
+        uses: actions/cache@v2
         with:
-          annotations-prefix: ${{ runner.os }}-${{ matrix.java-version }}/${{ matrix.maven-version }}
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Build Chain ${{ matrix.java-version }}
+        id: build-chain
+        uses: kiegroup/github-action-build-chain@v2.6.13
```
Comment (radtriste) [inline .github/workflows/kogito-examples-pr.yml:59]:
```suggestion
        uses: kiegroup/github-action-build-chain@v2.6.18
```

File: .github/workflows/kogito-runtimes-pr.yml [hunk 3]
```diff
@@ -22,30 +22,50 @@ jobs:
...
-          annotations-prefix: ${{ runner.os }}-${{ matrix.java-version }}/${{ matrix.maven-version }}
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-      - name: Surefire Report
-        uses: kiegroup/kogito-pipelines/.ci/actions/surefire-report@main
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Build Chain ${{ matrix.java-version }}
+        id: build-chain
+        uses: kiegroup/github-action-build-chain@v2.6.13
```
Comment (radtriste) [inline .github/workflows/kogito-runtimes-pr.yml:59]:
```suggestion
        uses: kiegroup/github-action-build-chain@v2.6.18
```

File: .github/workflows/optaplanner-pr.yml [hunk 4]
```diff
@@ -22,31 +22,51 @@ jobs:
...
+      - name: Cache Maven packages
+        uses: actions/cache@v2
         with:
-          annotations-prefix: ${{ runner.os }}-${{ matrix.java-version }}/${{ matrix.maven-version }}
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Build Chain ${{ matrix.java-version }}
+        id: build-chain
+        uses: kiegroup/github-action-build-chain@v2.6.13
```
Comment (radtriste) [inline .github/workflows/optaplanner-pr.yml:59]:
```suggestion
        uses: kiegroup/github-action-build-chain@v2.6.18
```

**Review by cristianonicolai — APPROVED** [review]

**Review by evacchi — APPROVED** [review]

**Comment by Ginxo** [comment]
due to `quarkus-bootstrap-maven-plugin` failure from https://github.com/kiegroup/kogito-runtimes/pull/1713

**Pull request merged**